### PR TITLE
test if no problem release it 

### PR DESCRIPTION
**Pull request by windatang**

### CODE REVIEW

**Comment by pavpd** [comment]
no problem, release it

**Comment by pavpd** [comment]
> _No description provided._

no problem, release it

**Comment by Tommy-ASD** [comment]
problem, continue hype until solved

File: packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx [hunk 1]
```diff
@@ -347,9 +350,9 @@ export const TransactionTable = withTranslation(['tables', 'common'])((props: Tr
         {
             key: 'status',
             name: t('labelTxMemo'),
-            headerCellClass: 'textAlignRight',
+            headerCellClass: 'textAlingCenter',
```
Comment (Maikell84) [inline packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx:353]: Typo.
```suggestion
            headerCellClass: 'textAlignCenter',
```

File: packages/common-resources/static-resources/src/i18n/en_US/common.ts [hunk 2]
```diff
@@ -177,7 +178,7 @@ export default {
     labelConnectBy: 'Connected with <1>{{connectBy}}</1>',
     labelWrongNetwork: 'Wrong network',
     labelActivatedAccountDeposit: 'Please deposit to activate your Layer2 account',
-    labelActivatedAccountNotSupport: 'We are apologize, Loopring app not support your address current now.',
+    labelActivatedAccountNotSupport: 'We apologize, loopring app does not support your address currently.',
```
Comment (Maikell84) [inline packages/common-resources/static-resources/src/i18n/en_US/common.ts:181]:
```suggestion
    labelActivatedAccountNotSupport: 'We apologize, loopring app does currently not support your address.',
```

File: README.md [hunk 3]
```diff
@@ -31,11 +31,13 @@ yarn up
...
-- Welcome to improve or create feature or loopring layer2 website, please follow and use the basic UI-Components,
-see it on component-lib.
-- Project separate a lib call web3-provider, it's a third-part ETH web3 wallet provider service (wallectConnect & metamask), 
-- Welcome to reuse it or support your provider service for our website. 
-- Suggestions and idea are welcome.
+- We appreciate your improvement or initiatives for loopring layer2 website, please view the source code in the component-lib.
+
+- Project separates a lib called web3-provider, which is a third-party ETH web3 wallet provider service (wallectConnect & metamask), 
+
+- Welcome to reuse it or integrate your provider service to our website.
```
Comment (Maikell84) [inline README.md:38]:
```suggestion
- You are welcome to reuse it or integrate your provider service to our website. 
```

**Review by Maikell84 — COMMENTED** [review]
Hi! Good work with this PR!
I have added some minor suggestions. Feel free to discuss!

**Comment by localyears** [comment]
wen release v2 so moon soon?🚀🚀🚀🚀

File: packages/common-resources/static-resources/src/i18n/zh_CN/common.ts [hunk 4]
```diff
@@ -177,7 +178,7 @@ export default {
     labelConnectBy: '当前连接 <1>{{connectBy}}</1>',
     labelWrongNetwork: '未识别网络',
     labelActivatedAccountDeposit: '首次开通二层钱包需充值并激活',
-    labelActivatedAccountNotSupport: 'We are apologize, Loopring app not support your address current now.',
+    labelActivatedAccountNotSupport: 'We apologize, loopring app does not support your address currently.',
```
Comment (atae) [inline packages/common-resources/static-resources/src/i18n/zh_CN/common.ts:181]: This is technically fine, but a more native sentence would probably sound like "Loopring does not currently support your address. We apologize for the inconvenience."

File: packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx [hunk 5]
```diff
@@ -112,9 +112,12 @@ const TableStyled = styled(Box)`
         .textAlignRight {
             text-align: right;
         }
-        .textAlingCenter {
+        .textAlignCenter {
             text-align: center;
         }
+        .textAlignLeft{
```
Comment (Krymonota) [inline packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx:118]:
```suggestion
        .textAlignLeft {
```

File: packages/webapp/src/hooks/useractions/useNFTTransfer.ts [hunk 6]
```diff
@@ -39,7 +39,7 @@ import { useBtnStatus } from 'hooks/common/useBtnStatus';
 import { useModalData } from 'stores/router';
 import { isAccActivated } from './checkAccStatus';
 import { getFloatValue } from 'utils/formatter_tool';
-import { UserNFTBalanceInfo } from '@loopring-web/loopring-sdk';
+    import { ErrorMsg, UserNFTBalanceInfo } from '@loopring-web/loopring-sdk';
```
Comment (Krymonota) [inline packages/webapp/src/hooks/useractions/useNFTTransfer.ts:42]:
```suggestion
import { ErrorMsg, UserNFTBalanceInfo } from '@loopring-web/loopring-sdk';
```

File: packages/webapp/src/hooks/useractions/useNFTTransfer.ts [hunk 7]
```diff
@@ -219,9 +219,9 @@ export const useNFTTransfer = <R extends IBData<T> & Partial<NFTTokenInfo & User
 
                 if (isAccActivated()) {
 
-                    if (response?.errorInfo) {
+                    if ((response as sdk.ErrorMsg)?.errMsg ) {
```
Comment (Krymonota) [inline packages/webapp/src/hooks/useractions/useNFTTransfer.ts:222]:
```suggestion
                    if ((response as sdk.ErrorMsg)?.errMsg) {
```

File: packages/webapp/src/hooks/useractions/useNFTWithdraw.ts [hunk 8]
```diff
@@ -271,7 +271,7 @@ export const useNFTWithdraw = <R extends IBData<T> & Partial<NFTTokenInfo & User
                         } else {
                             setShowAccount({ isShow: true, step: AccountStep.Withdraw_Failed })
                         }
-                    } else if (response?.resultInfo) {
+                    } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo ) {
```
Comment (Krymonota) [inline packages/webapp/src/hooks/useractions/useNFTWithdraw.ts:274]:
```suggestion
                    } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo) {
```

File: packages/webapp/src/hooks/useractions/useTransfer.ts [hunk 9]
```diff
@@ -211,7 +211,7 @@ export const useTransfer = <R extends IBData<T>, T>(): {
                         } else {
                             setShowAccount({ isShow: true, step: AccountStep.Transfer_Failed })
                         }
-                    } else if (response?.resultInfo) {
+                    } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo ) {
```
Comment (Krymonota) [inline packages/webapp/src/hooks/useractions/useTransfer.ts:214]:
```suggestion
                    } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo) {
```

File: packages/webapp/src/hooks/useractions/useWithdraw.ts [hunk 10]
```diff
@@ -271,7 +271,7 @@ export const useWithdraw = <R extends IBData<T>, T>(): {
                         } else {
                             setShowAccount({ isShow: true, step: AccountStep.Withdraw_Failed })
                         }
-                    } else if (response?.resultInfo) {
+                    } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo ) {
```
Comment (Krymonota) [inline packages/webapp/src/hooks/useractions/useWithdraw.ts:274]:
```suggestion
                    } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo) {
```

File: packages/webapp/src/modal/AccountModal/hook.tsx [hunk 11]
```diff
@@ -81,7 +81,7 @@ export function useAccountModalForUI({t, etherscanBaseUrl, onClose, rest, }:
         modals: {isShowAccount}, setShowConnect, setShowAccount,
         setShowDeposit, setShowTransfer, setShowWithdraw, setShowResetAccount,
     } = useOpenModals()
-
+    const {lastStep} =  useModalData();
```
Comment (Krymonota) [inline packages/webapp/src/modal/AccountModal/hook.tsx:84]:
```suggestion
    const {lastStep} = useModalData();
```

File: packages/webapp/src/services/account/activateAccount.ts [hunk 12]
```diff
@@ -103,40 +103,38 @@ export async function updateAccountFromServer({isHWAddr, feeInfo, isReset, }: {
...
-                        if (updateAccountResponse.errorInfo) {
+                        if ((response as sdk.ErrorMsg)?.errMsg) {
                             result.code = ActionResultCode.UpdateAccoutError
                             result.data = {
                                 eddsaKey,
-                                errorInfo: updateAccountResponse.errorInfo,
+                                ...response,
                             }
-                        } else if (updateAccountResponse?.resultInfo) {
+                        } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo ) {
```
Comment (Krymonota) [inline packages/webapp/src/services/account/activateAccount.ts:122]:
```suggestion
                        } else if ((response as sdk.TX_HASH_RESULT<sdk.TX_HASH_API>)?.resultInfo) {
```

**Review by Krymonota — COMMENTED** [review]
Hi! Just a few insignificant spacing / formatting issues 🙂

File: packages/component-lib/src/components/block/AssetTitle.tsx [hunk 13]
```diff
@@ -56,6 +58,10 @@ export const AssetTitle = withTranslation('common')(({
             </Typography>
         </Grid>
         <ButtonListRightStyled item xs={5} display={'flex'} flexDirection={'row'} justifyContent={'flex-end'}>
+            {/* <Button variant={'outlined'} size={'medium'} color={'primary'} style={{ minWidth: 120, textTransform: 'none' }}
+                onClick={showRamp}>
+                {t('labelAssetsBtnRamp')}
+            </Button> */}
```
Comment (brandonnorsworthy) [inline packages/component-lib/src/components/block/AssetTitle.tsx:64]: Remove commented elements that were used for testing as these can end up here forever

File: packages/common-resources/static-resources/src/i18n/en_US/common.ts (hunk 2 above)
Comment (FuzzyBearBTC) [reply packages/common-resources/static-resources/src/i18n/en_US/common.ts:181]: We apologize, loopring app currently does not support your address.

File: packages/component-lib/src/components/tradePanel/components/AmmWrap/AmmWithDraw.tsx [hunk 14]
```diff
@@ -1,348 +1,348 @@
...
+
+  const showPercentage = _selectedPercentage < 0 || _selectedPercentage > 100 ? EmptyValueTag + '%' : `${_selectedPercentage}%`
+  const lpTradeValue = ammData?.coinLP?.tradeValue
+  let lpBalance: any = ammData?.coinLP?.balance
+  lpBalance = parseFloat(lpBalance)
+  const showLP = (lpBalance && lpTradeValue && lpTradeValue > 0 && lpTradeValue <= lpBalance) ? getValuePrecisionThousand(lpTradeValue, 2, 6) : '0'
+
+  const miniA = ammData?.coinA?.tradeValue ? getValuePrecisionThousand(ammData?.coinA?.tradeValue) : EmptyValueTag
+
+  // myLog('ammData?.coinB?.tradeValue:', ammData?.coinB?.tradeValue, getValuePrecisionThousand(ammData?.coinB?.tradeValue))
```
Comment (FuzzyBearBTC) [inline packages/component-lib/src/components/tradePanel/components/AmmWrap/AmmWithDraw.tsx:172]: Can this commented out code be removed?

File: packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx (hunk 1 above)
Comment (windatang) [reply packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx:353]: the column is alignCenter or not if it's AlignRight keep it as Right

Comment (GuillaumeBenini) [reply packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx:353]: He is referring to the typo "Aling", correcting with "Align" instead.

File: static/documents/markdown/plugin_guide.md [hunk 15]
```diff
@@ -1,20 +1,21 @@
-## How To Disable Plugins/Add-Ons/Extensions In Multiple Browsers
+# How To Disable Plugins/Add-Ons/Extensions In Multiple Browsers
 
-### Mozilla Firefox:
+### **Mozilla Firefox**:
```
Comment (brandonnorsworthy) [inline static/documents/markdown/plugin_guide.md:3]: can you change this line to ```### **Mozilla Firefox:**``` to match the other header 3 changes in this guide

**Review by brandonnorsworthy — APPROVED** [review]
nice work on these adjustments I added one suggestion can you take a look?

**Review by markjang123 — APPROVED** [review]
lgtm!

**Review by SylvanasGone — APPROVED** [review]

**Comment by localyears** [comment]
release what tho

**Pull request merged**